### ISSUE TITLE
[Health 9.0] can't be installed in flutter 3.16 stable due to intl incompatibility 

### ISSUE THREAD

**Issue by dgilperez** [cph-cachet/flutter-plugins#886]
### Describe the bug

Upgrading Health to 9.0 is not possible with flutter 3.16.9, latest stable, when using `flutter_localizations`. Health is now depending on `intl` `0.19`, while `flutter_localizations` depends on the flutter sdk, that depends on `0.18.1`

Full message: 

```
Resolving dependencies...
Note: intl is pinned to version 0.18.1 by flutter_localizations from the flutter SDK.
See https://dart.dev/go/sdk-version-pinning for details.


Because x depends on flutter_localizations from sdk which depends on intl 0.18.1, intl 0.18.1 is required.
So, because x depends on intl ^0.19.0, version solving failed.
exit code 1
```

Was this dependency upgrade needed? Is it possible to get it back to 0.18.1?

**Comment by dgilperez** [comment]
This is an open issue on flutter by the way https://github.com/flutter/flutter/issues/139861

**Comment by dgilperez** [comment]
I am going to test a dependency override in the meanwhile. I will report if this does not work, for reference.

**Comment by NeKoFu** [comment]
override to force the version works well for me.
```
dependency_overrides: 
  intl: ^0.19.0
```

**Comment by bardram** [comment]
Have released a version 9.01 with

 ` intl: '>=0.18.0 <1.0.0' ``
